src/renderer/dx/lib/dx.vcxproj

```diff
@@ -21,6 +21,7 @@
     <ClCompile Include="..\precomp.cpp">
       <PrecompiledHeader>Create</PrecompiledHeader>
     </ClCompile>
+    <ClCompile Include="..\DxFontRenderData.cpp" />

```

Comment:
this will need to go into `sources` or `sources.inc` as appropriate as well.

Reply:
huh I thought only conhost-related thing need to be in sources.inc because of the OS build. Will do.